packages/components/src/progress-bar/index.tsx

```diff
@@ -44,6 +44,18 @@ function UnforwardedProgressBar(
 	);
 }
 
+/* A simple horizontal progress bar component.
+ *
+ * Supports two modes: determinate and indeterminate. A progress bar is determinate when a specific progress value has been specified (from 0 to 100), and indeterminate when a value hasn't been specified.
+ *
+ * ```jsx
+ * import { ProgressBar } from '@wordpress/components';
+ *
+ * const MyLoadingComponent = () => {
+ *  return <ProgressBar />
+ * }
```

Comment:
Let's fix this syntax as well:

```suggestion
 * const MyLoadingComponent = () => {
 * 	return <ProgressBar />;
 * };
```